### PR TITLE
Display the release command output if it is available

### PR DESCRIPTION
**Pull request by dmathieu**
In the case of a promotion with a single target, we will fetch the release, look for `output_stream_url` and, if it is present, stream the data it returns.

@heroku/devex this is to start discussion. You might have a better implementation.

cc @heroku/build

### CODE REVIEW

File: commands/pipelines/promote.js [hunk 1]
```diff
@@ -156,8 +197,9 @@ module.exports = {
       heroku, promotionActionName, coupling.pipeline.id, coupling.app.id, targetApps
     )
 
-    const pollLoop = pollPromotionStatus(heroku, promotion.id)
-    const promotionTargets = yield cli.action('Waiting for promotion to complete', pollLoop)
+    const pollLoop = pollPromotionStatus(heroku, promotion.id, true)
+    var promotionTargets = yield cli.action('Waiting for promotion to complete', pollLoop)
```
Comment (appleton) [inline commands/pipelines/promote.js]: This is so minor that I almost wouldn't bother commenting but could this be a `let` instead of `var` please?

Comment (dmathieu) [inline commands/pipelines/promote.js]: :+1:

**Comment by dmathieu** [comment]
@gudmundur @appleton I've added a test.

**Comment by dmathieu** [comment]
And I've adapted this PR to properly stream the data, and mock the output using `std-mock`, as we do in heroku/heroku-apps/pull/164

**Pull request merged**